app/models/user.rb

```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
-  devise :database_authenticatable,
-    :registerable, :jwt_authenticatable,
-    jwt_revocation_strategy: JwtDenylist
+  devise :database_authenticatable, :recoverable, 
+  :registerable, :jwt_authenticatable, 
+  jwt_revocation_strategy: JwtDenylist
```

Comment:
Let's keep the indentation here
```suggestion
    :registerable, :jwt_authenticatable, 
    jwt_revocation_strategy: JwtDenylist
```